tests/test_TF_determinismus.py

```diff
@@ -0,0 +1,66 @@
+# tests if the graph values of two different training runs on GPU are equal
+# make sure GPU is available
+# in some cases the test will give a false negative
+
+import sys
+sys.path += ["."]  # Python 3 hack
+
+from TFEngine import *
+from Config import Config
+
+from numpy.testing import assert_array_equal, assert_equal
+
+config = Config()
+config.load_file("demos/demo-tf-vanilla-lstm.12ax.config")
+
+
+def load_data():
+  from rnn import load_data
+  dev_data, _ = load_data(config, 0, 'dev', chunking="200:200", seq_ordering="sorted", shuffle_frames_of_nseqs=0)
+  eval_data, _ = load_data(config, 0, 'eval', chunking="200:200", seq_ordering="sorted", shuffle_frames_of_nseqs=0)
+  train_data, _ = load_data(config, 0, 'train')
+  return dev_data, eval_data, train_data
+
+
+def test_determinism_of_vanillalstm():
```

Comment:
Add a docstring for each test, and document your results on the test. Sth like this:
"With GPU (or CPU), TF x.y, this runs deterministic."